README.md

```diff
@@ -26,6 +26,7 @@ Board                          |  Connectivity     | Storage for credentials and
 `Embedded Planet EP_AGORA`          | Cellular          | SPIF                    | Build-only
 `ST NUCLEO_H743ZI2`                 | Ethernet          | Internal Flash          | Build-only
 `ST NUCLEO_L4R5ZI`                  | Wi-Fi ESP8266     | Internal Flash          | Build-only
+`Nuvoton NUMAKER_IOT_M487`          | Wi-Fi ESP8266     | SD card (NUSD)          |
```

Comment:
This could have Build-only until we have CI-support in place.

Reply:
@teetak01 Changed to build-only

Reply:
Thanks